client/src/test/java/model/DataProvider.java

```diff
@@ -39,4 +65,18 @@
     return coordinates;
   }
 
+  @org.testng.annotations.DataProvider(name = "positions")
+  public static Object[][] positions() {
+
+    int testCount = 10;
+    int boundForRandom = 99;
+    Object[][] positionsIndexes = new Object[testCount][];
+
+    for(int i = 0; i < testCount; i++) {
+      Random random = new Random();
```

Comment:
I my opinion random values shouldn't be used in test

Reply:
This is good point, I think we should ask Artem about that